credentials/settings/base.py

```diff
@@ -434,3 +435,8 @@
         'debug_toolbar.panels.redirects.RedirectsPanel',
     ]
 # END DJANGO DEBUG TOOLBAR CONFIGURATION
+
+# Edly user info cookie specific values
+EDLY_USER_INFO_COOKIE_NAME = 'edly-user-info'
+EDLY_COOKIE_SECRET_KEY = 'EDLY-COOKIE-SECRET-KEY'
```

Comment:
@iam-mhaseeb Please create a subtask for devops to override `EDLY_COOKIE_SECRET_KEY` in credentials env, similar to ECOM and LMS

CC: @jamshedali20

Reply:
Done